logging/access.go

```diff
@@ -17,8 +18,8 @@ const (
 	// format:
 	// remote_host - - [date] "method uri protocol" status response_size "referer" "user_agent"
 	combinedLogFormat = commonLogFormat + ` "%s" "%s"`
-	// We add the duration in ms, a requested host and a flow id
-	accessLogFormat = combinedLogFormat + " %d %s %s\n"
+	// We add the duration in ms, a requested host and a flow id and audit log
+	accessLogFormat = combinedLogFormat + " %d %s %s %s\n"
```

Comment:
if we do it so, then there will be a trailing space in the access log of those deployments that don't use this field. there's a chance that some already used parsers may consider this extra trailing space as part of the flow id. I would play it safe and use the new format only when there is a 'X-Unverified-Audit' header found.

Reply:
This will cause the log format to flip based on whether the request is authenticated or not, I'm not sure if it's a good idea. If keeping backwards compatibility is a must, then the only thing that can be done is a command-line argument, otherwise I'd just flip the order of flow id and sub claim and hope that most of the parsers can deal with an extra value. This will of course require missing flow ids to be represented as "-" instead of just a space, but it's probably a good idea anyway.

Reply:
@aermakov-zalando see the output of https://github.com/zalando/skipper/pull/637#issuecomment-385473261 all whitespaced fields will be replaced by "-" and we agreed on having this change a fix, which was overseen and a bug before.

Reply:
Then just swapping it should be enough, I guess.